proposals/0029-cooperative-vector.md

```diff
@@ -332,36 +349,98 @@ can be found in [Minimum Support Set].
 
 ### Type Interpretations
 
-The various "interpretation" arguments specify a value from the following enum:
+The `ComponentType` enum in `DxilConstants.h` is extended as shown below, with
```

Comment:
Do we want a file reference in the spec? isn't the enumeration alone sufficient?

Reply:
Learning about the location of these enums was pretty key to be understanding parts of the discussion around this, so I think it is worth including to be clear what we mean when we list enums.